base_station_nodes: &id_base_stations
  - WayPoint140
#  - WayPoint141
wait_nodes: *id_base_stations
local_storage_nodes:
  - WayPoint68
cold_storage_node: None
use_cold_storage: false
charging_station_nodes:
  - WayPoint72
robot_ids: 
  - thorvald_014
#  - thorvald_019
max_task_priorities: 
  - 255
#  - 255
picker_ids:
  - picker_b827ebf2dedb
  - picker_b827ebe5575f
  - picker_000000000000
  - picker01
  - picker02
  - picker03
  - picker04
  - picker05
  - picker06
  - picker07
  - picker08
  - picker09
  - picker10
virtual_picker_ids:
  - picker101
  - picker102
  - picker103
  - picker104
  - picker105
  - picker106
  - picker107
  - picker108
  - picker109
  - picker110
# is this a simulated environment
use_sim: false
# if batter_voltage is below the given threshold, the robot is set as unfit. ignored for sim
low_battery_voltage: 45.0
